packages/language-support/package.json

```diff
@@ -14,11 +22,11 @@
   },
   "scripts": {
     "gen-parser": "antlr4 -Dlanguage=TypeScript -visitor src/antlr-grammar/CypherParser.g4 src/antlr-grammar/CypherLexer.g4 -o src/generated-parser/ -Xexact-output-dir",
-    "postinstall": "npm run gen-parser",
```

Comment:
We don't need this as far as I know. The generated parser should be included in the package